horace_core/sqw/@SQWDnDBase/sqw_eval_nopix.m

```diff
@@ -18,4 +18,3 @@
     obj(i).data = obj(i).data.sqw_eval_nopix(sqwfunc, all_bins, pars);
 end
 
-end
```

Comment:
Noise. Should not be in PR. Also by MATLAB recommendations, functions should finish with `end`